svg-2-android-vector/src/main/java/com/quittle/svg2androidvector/PreBuildDependencyAdder.java

```diff
@@ -0,0 +1,27 @@
+package com.quittle.svg2androidvector;
+
+import com.android.build.gradle.api.BaseVariant;
+
+import org.gradle.api.Action;
+import org.gradle.api.Task;
+import org.jetbrains.annotations.NotNull;
```

Comment:
Could you not add the dependency on jetbrains? I don't believe this package is guaranteed to be available as it isn't provided by the Gradle runtime environment.

Reply:
Sure, that's just a static-analysis annotation.

Reply:
fixed.